docs/ides/configuring-web-ides.md

```diff
@@ -117,30 +132,86 @@ As a starting point, see the [projector-container](https://github.com/bpmct/code
 - WebStorm
 - ➕ code-server (just in case!)
 
-## Custom IDEs and applications
+## JupyterLab
 
-As long as the process is running on the specified port inside your resource, you support any application.
+Configure your agent and `coder_app` like so to use Jupyter:
 
-```sh
-# edit your template
-cd your-template/ 
-vim main.tf
+```hcl
+data "coder_workspace" "me" {}
+
+resource "coder_agent" "coder" {
+  os   = "linux"
+  arch = "amd64"
+  dir  = "/home/coder"
+  startup_script = <<-EOF
+pip3 install jupyterlab
+jupyter lab --ServerApp.base_url=/@${data.coder_workspace.me.owner}/${data.coder_workspace.me.name}/apps/Jupyter/ --ServerApp.token='' --ip='*'
+EOF
+}
+
+resource "coder_app" "Jupyter" {
```

Comment:
Let's make this `j` lowercase for now. It'll have to be lowered in both URLs as well.